### ISSUE TITLE
Services getting created with space in the service_id

### ISSUE THREAD

**Issue by kaizersoje** [jeroenterheerdt/HADailySensor#14]
I created a sensor "Garage Door Daily Count"

![image](https://user-images.githubusercontent.com/54566217/128439064-b0e4cf6c-bbeb-4308-bff7-473892305fa9.png)

The services got created with spaces.

```
daily.garage door daily count_update
daily.garage door daily count_reset
```

The services cannot be used in automations if there are spaces.

**Comment by jeroenterheerdt** [comment]
huh, that's weird. Let me check what is going on here.

**Comment by jeroenterheerdt** [comment]
thanks for reporting, can you please install this beta release: https://github.com/jeroenterheerdt/HADailySensor/releases/tag/v0.3.3beta and see if it helps?

**Comment by jeroenterheerdt** [comment]
please report back if it improves the situation and does not result in errors

**Comment by kaizersoje** [comment]
Yes it's resolved in v0.3.3b. Thx.

**Comment by jeroenterheerdt** [comment]
thanks!
